src/monodroid/jni/monodroid-networkinfo.c

```diff
@@ -128,11 +135,11 @@ _monodroid_get_network_interface_supports_multicast (const char *ifname, mono_bo
 MONO_API int
 _monodroid_get_dns_servers (void **dns_servers_array)
 {
-	*dns_servers_array = NULL;
```

Comment:
I can just imagine the C++ optimizer seeing this and going "`dns_servers_array` can't be null! therefore the following `if` + `return -1` is dead code!"...

Scary.

Reply:
yep...